src/components/user/education/education.edgedb.repository.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { ID, NotFoundException, PublicOf } from '~/common';
+import { e, RepoFor } from '~/core/edgedb';
+import { Education } from './dto';
+import { EducationRepository } from './education.repository';
+
+@Injectable()
+export class EducationEdgeDBRepository
+  extends RepoFor(Education, {
+    hydrate: (education) => education['*'],
+  }).withDefaults()
+  implements PublicOf<EducationRepository>
+{
+  async getUserIdByEducation(id: ID) {
+    const edu = e.cast(e.User.Education, e.uuid(id));
+    const query = e.select(e.User, (user) => ({
+      id: true,
```

Comment:
@CarsonF - Is this `id` necessary?  Wouldn't that just be the default?

Reply:
Not sure, try removing lol